src/api.js

```diff
@@ -70,7 +69,7 @@ export default class Api {
      * The even emitter instance.
      * @type {EventEmitter}
      */
-    this.events = options.events || new EventEmitter();
+    this.events = options.events;
```

Comment:
I don't consider a change to the API of `Api` really a Kinto API change. ;) It just means that instead of providing the default in `api.js`, we provide it in `index.js`, which is indeed kind of nice because it means the constructor has less active code.

(unrelated to this PR) I think the code comment `- {String}       events  The HTTP request mode.` on line 40/39 should I think be `- {String}       requestMode  The HTTP request mode.`